src/shared-components/posts/createPostModal/createPostModal.module.css

```
@@ -1,159 +1,73 @@
-  z-index: 1050;
-}
-
-/* Header Styles */
-.modalHeader {
-  display: flex;
-  justify-content: space-between;
-  align-items: flex-start;
-  padding: var(--space-5) var(--space-7) var(--space-4) var(--space-5);
-  border-bottom: none;
+/* Modal customization for CRUDModalTemplate */
+.createPostModal :global(.modal-dialog) {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)